branchpro/_incidence_number_plot.py

```diff
@@ -0,0 +1,81 @@
+#
+# IncidenceNumberPlot Class
+#
+# This file is part of BRANCHPRO
+# (https://github.com/SABS-R3-Epidemiology/branchpro.git) which is released
+# under the BSD 3-clause license. See accompanying LICENSE.md for copyright
+# notice and full license details.
+#
+
+import pandas as pd
+import plotly.graph_objs as go
+
+
+class IncidenceNumberPlot():
+    """IncidenceNumberPlot Class
+    Stores the main figure for the Dash app.
+    """
+    def __init__(self):
+        self._figure = go.Figure()
+
+    def add_data(self, df, time_key='Time', inc_key='Incidence Number'):
+        """
+        Supplies data to the figure which will be used for the bar plot.
+
+        Parameters
+        ----------
+        df
+            (pandas DataFrame) contains numbers of new cases by days.
+        time_key
```

Comment:
For time_key and inc_key, I would add that these are the column names expected in the pandas dataframe